Yoast/wordpress-seo
# Get rid of JQMIGRATE warnings.

Very minor issue, but these JQMIGRATE warnings bother and there's no reason why we shouldn't get rid of them.
```
JQMIGRATE: 'ready' event is deprecated
JQMIGRATE: 'hover' pseudo-event is deprecated, use 'mouseenter mouseleave'
```

To test the first one:
- go in a plugin settings page, for example "Titles & Metas" `admin.php?page=wpseo_titles`
- click on a tab, for example "Homepage"
- the URL changes to `admin.php?page=wpseo_titles#top#home`
- refresh the page
- check in the browser DOM inspector the form with id `wpseo-conf` has the action attribute that ends with `#top#home`

To test the second one:
- go in the WordPress Reading Settings
- check the "Discourage search engines from indexing this site" checkbox to make the "new seo issue" notification bubble appear
- at this point you should be fast enough to over with your mouse either the bubble itself or any of the toolbar top level menu items: when hovering, the bubble should disappear (keep in mind it disappears anyways after 3 seconds)

<img width="658" alt="screen shot 2017-07-24 at 12 06 10" src="https://user-images.githubusercontent.com/1682452/28519577-6edda4a8-706c-11e7-88b0-1123d6fd6ae8.png">

Fixes #7142
